.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    steps:
+    - checkout
+
+    - run: |
+        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./architect
+        ./architect version
+    - run: ./architect build
+    - deploy:
```

Comment:
It would be good to start having a separate job for deploying that is run after e2e on master, as in https://github.com/giantswarm/chart-operator/blob/master/.circleci/config.yml. Not a blocker, we can iterate on this later.

Reply:
Ack. Will consider this in the next PRs.